packages/core/content-manager/server/src/services/document-metadata.ts

```diff
@@ -85,7 +85,17 @@ export default ({ strapi }: { strapi: Strapi }) => ({
       const draftVersion = localeVersions.find((v) => v.publishedAt === null);
       const otherVersions = localeVersions.filter((v) => v.id !== draftVersion?.id);
 
-      if (!draftVersion) return;
+      if (!draftVersion) {
+        if (otherVersions.length === 1) {
+          // If there is no draft version, draft and publish is disabled
+          return {
+            ...pick(AVAILABLE_LOCALES_FIELDS, otherVersions[0]),
+            status: this.getStatus(otherVersions[0]),
+          };
+        }
+
+        return;
+      }
```

Comment:
Should we throw an error at L97 in this case?

Reply:
In case someone broke the rule of always having a draft, we still want the content manager to be functional. So I would avoid throwing errors if that happen. I think returning nothing should be fine here